### PR TITLE
view table with html

### PR DESCRIPTION
**Pull request by dekespo**
Html file is generated as "resutls.html". It can be viewed with any browser. It has a bit style and the style can be changed anytime :) .

### CODE REVIEW

File: benchmarker.py [hunk 1]
```diff
@@ -100,10 +100,22 @@ def save_to_file(results):
...
+		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str(round(next, 3)) + " s" + "</td>" for next in results[i]])
+		html += "<tr>" + row + "</tr>"
+	html += "</html></body></table>"
+	htmlFile = open("results.html", "w")
+	htmlFile.write(html)
+	htmlFile.close()
 
 def main():
 	results = benchmark()
 	save_to_file(results)
```
Comment (naslundx) [inline benchmarker.py:117]: Ah, great that you kept this, we will use it for text file generation.

**Comment by naslundx** [comment]
@dekespo You are becoming a master.

**Comment by naslundx** [comment]
I've got nothing to comment on, nice.

**Comment by dekespo** [comment]
Hahaha :D , thanks a lot. Actually there is a tiny bit error on the line 110 the escapes /table and /html should be swapped. It can be done in the next commit.

**Pull request merged**